cc.spek.Spek.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+    <name>Spek</name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0-only</project_license>
+    <summary>Acoustic Spectrum Analyser</summary>
+    <description>
+        <p>Spek is an acoustic spectrum analyser written in C++. It uses FFmpeg libraries for audio decoding and wxWidgets for the GUI.</p>
+    </description>
+    <developer id="www.spek.cc">
+        <name>Alexander Kojevnikov</name>
+    </developer>
+    <update_contact>joshua.gage.stone@gmail.com</update_contact>
+    <url type="homepage">https://www.spek.cc/</url>
+    <url type="bugtracker">https://github.com/alexkay/spek/issues</url>
+    <url type="translate">https://explore.transifex.com/spek/spek/</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://substack-post-media.s3.amazonaws.com/public/images/87fd2840-4ee7-492e-b15f-38a384a56a4c_724x590.png</image>
+        </screenshot>
+    </screenshots>
+    <content_rating type="oars-1.1">
+      <content_attribute id="violence-cartoon">none</content_attribute>
+      <content_attribute id="violence-fantasy">none</content_attribute>
+      <content_attribute id="violence-realistic">none</content_attribute>
+      <content_attribute id="violence-bloodshed">none</content_attribute>
+      <content_attribute id="violence-sexual">none</content_attribute>
+      <content_attribute id="violence-desecration">none</content_attribute>
+      <content_attribute id="violence-slavery">none</content_attribute>
+      <content_attribute id="violence-worship">none</content_attribute>
+      <content_attribute id="drugs-alcohol">none</content_attribute>
+      <content_attribute id="drugs-narcotics">none</content_attribute>
+      <content_attribute id="drugs-tobacco">none</content_attribute>
+      <content_attribute id="sex-nudity">none</content_attribute>
+      <content_attribute id="sex-themes">none</content_attribute>
+      <content_attribute id="sex-homosexuality">none</content_attribute>
+      <content_attribute id="sex-prostitution">none</content_attribute>
+      <content_attribute id="sex-adultery">none</content_attribute>
+      <content_attribute id="sex-appearance">none</content_attribute>
+      <content_attribute id="language-profanity">none</content_attribute>
+      <content_attribute id="language-humor">none</content_attribute>
+      <content_attribute id="language-discrimination">none</content_attribute>
+      <content_attribute id="social-chat">none</content_attribute>
+      <content_attribute id="social-info">none</content_attribute>
+      <content_attribute id="social-audio">none</content_attribute>
+      <content_attribute id="social-location">none</content_attribute>
+      <content_attribute id="social-contacts">none</content_attribute>
+      <content_attribute id="money-purchasing">none</content_attribute>
+      <content_attribute id="money-gambling">none</content_attribute>
+    </content_rating>
```

Comment:
I don't see releases here.

Reply:
The releases information is generated through NEWS via `appstreamcli news-to-metainfo --format=markdown NEWS ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml`